all.proto

```diff
@@ -1,5 +1,4 @@
 // This file is for debugging purposes
 // Tools like BloomRPC and other CLIs have problems with imports
-import public "api/v1/article/article.proto";
-import public "api/v1/article/service.proto";
-import public "api/v1/section/service.proto";
```

Comment:
Warum gibt es section nicht mehr? Ich fand die Trennung durch aus sinnvoll. AMP braucht keine sections

Reply:
we will have a separate section api! Folder is there, just need the IDL!